common/build/eslint-config-fluid/src/custom-rules/no-release-tags.js

```diff
@@ -0,0 +1,54 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const { TSDocParser } = require("@microsoft/tsdoc");
+
+function hasReleaseTag(comment) {
+	const parser = new TSDocParser();
+	const parserContext = parser.parseString(comment);
+
+	const hasReleaseTag =
+		parserContext.docComment.modifierTagSet.isAlpha() ||
+		parserContext.docComment.modifierTagSet.isBeta() ||
+		parserContext.docComment.modifierTagSet.isPublic() ||
+		parserContext.docComment.modifierTagSet.isInternal();
+
+	return hasReleaseTag;
+}
+
+module.exports = {
```

Comment:
Note: writing a fixer for this rule should also be possible (i.e. just remove the release tag from the item), but I would suggest we file a separate task for that and do it later.